hbase-protocol-shaded/src/main/protobuf/Backup.proto

```diff
@@ -54,11 +54,12 @@ message TableServerTimestamp {
 
 /**
  * Structure keeps relevant info for backup restore session
+ * backup_root_dir is deprecated because we no longer store root dirs on disk HBASE-28882
```

Comment:
We have a kind of standard language to decorate code that is participating in a deprecation cycle.

> This foo was marked as deprecated in HBase x.y.z, will be removed in (x+2).0.0.